build.gradle

```diff
@@ -24,7 +24,7 @@ allprojects {
 
     }
 
-    project.version = "1.0.0-beta09"
+    project.version = "1.0.0-beta09-SNAPSHOT"
 
 }
```

Comment:
this should be 1.0.0-beta10-SNAPSHOT,
as we had mentioned, we have released "1.0.0-beta09" already
https://mvnrepository.com/artifact/com.hyperwallet.android.ui/userrepository.  please update the version for other component's grade file as well.